README.md

```diff
@@ -0,0 +1,116 @@
+### FILES ORGANIZATION
+---
+##### data [This folder contains the data files for the simulator]
+1. input
+    -   input_abp_0.txt
+    -    input_abp_1.txt
+2. output
+	-	-abp_output_0.txt
+	-	-abp_output_1.txt
+
+##### doc [This folder contains the documentation for the project]
+1. alternatebitprot.pdf
+2. Cadmium_Documentation_Ubuntu.pdf
+3. Cadmium_Windows.pdf
+
+##### include[This folder contains the header files]
+1. message.hpp
+2. receiver_cadmium.hpp
+3. sender_cadmium.hpp
+4. subnet_cadmium.hpp
+
+##### lib [This folder contains 3rd party libraries needed in the project]
+1. cadmium-master[This folder contains cadmium library files]
+2. DESTimes-master[This folder contains DesTimes library files]
+3. vendor[this folder contains third party vendor files]
+	-	iestream.hpp
+	-	nd_time.hpp
+
+##### src [This folder contains the source files written in c++ for the project]
+1. main.cpp
+2. message.cpp
+
+##### test [This folder the unit test for the different include files]
+1. data [This folder contains the data files for test folder]
+	1.1.receiver
+	-	receiver_input_test.txt
+	-	receiver_test_output.txt
+    
+    1.2.sender
+	-	sender_input_test_ack_In.txt
+	-   sender_input_test_control_In.txt
+	-   sender_test_output.txt
+		
+    1.3.subnet
+	-   subnet_input_test.txt
+	-   subnet_test_output.txt
+2. src [This folder contains the source files written in c++ for test folder]
+	2.1.receiver
+    -   main.cpp
+
+    2.2.sender
+	-   main.cpp
+	
+	2.3.subnet
+	-   main.cpp
+
+### STEPS TO RUN THE SIMULATOR
+---
+Document alternatebitprot.pdf in the doc folder contains the explanation of this simulator
+
+1.Update include paths in all the makefiles in this folder and subfolders. You need to update the following lines:
+
+1. INCLUDECADMIUM=-I lib/cadmium-master/include
+2. Update the relative path to cadmium/include from the folder where the makefile is. You need to take into account where you copied the folder during the installation process
+>               Example: INCLUDECADMIUM=-I lib/cadmium-master/include
+
+2.Run the unit tests
+
+2.1. Run subnet test
+1. Open the terminal.
+2. To compile the test, type in the terminal:
+>               make clean; make comp; make all
+3. To run the test set the command prompt in the bin folder. To do so, type in the terminal the path to this folder.
+>               Example: cd bin
+4. Once inside the bin folder, type in the terminal **"./NAME_OF_THE_COMPILED_FILE"**. For this specific test you need to type:
+>               ./SUBNET_TEST
+5. To check the output of the test, open  **"../test/data/subnet_test_output.txt"**
+
+2.2. To run receiver and sender tests, the steps are analogous to 2.1
+
+3 - Run the simulator
```

Comment:
for point 1, 2, there is dot following the number and then the description but for point 3, there is a hyphen after the number and then its description. Please correct to follow any one type of bullets throughout the file.

Reply:
Solved